libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ParentFiltersPage.tsx

```diff
@@ -129,12 +146,31 @@ export const ParentFiltersPage: React.FC<IParentFiltersPageProps> = ({
     attributeTitle,
     parentFilters,
     validParentFilters,
+    dependentDateFilters,
+    availableDatasets,
     onSelect,
     onGoBack,
     onClose,
+    onCommonDateSelect,
 }) => {
     const intl = useIntl();
-    const items = useFilterItems(parentFilters, validParentFilters);
+    const isEnabledKDAttributeFilterDatesValidation = useDashboardSelector(
+        selectEnableKDAttributeFilterDatesValidation,
+    );
+
+    let items = useFilterItems(parentFilters, validParentFilters);
+
+    const dependentDateItemsWithDimensions = useDateFilterItemsWithDimensions(
+        dependentDateFilters,
+        availableDatasets,
+        false,
+        intl,
+    );
+
+    if (isEnabledKDAttributeFilterDatesValidation) {
+        items = [...items, ...dependentDateItemsWithDimensions];
+    }
```

Comment:
It would be better to hide all this logic into `useFilterItems` hook and leave the component body mostly without change.

Reply:
Moved.